src/pages/overview/index.md

```diff
@@ -53,16 +53,16 @@ The PDF Extract API can be embedded into any application using the [PDFServices
-
-PDF Accessibility Auto-Tag API is a powerful tool that may be used as part of the process of improving accessibility in individual PDFs and can be applied at scale for large backlogs of PDFs. Auto-tagging adds document structure tags to the PDF file that are used to read a document's text and present it in a way that makes sense to users using assistive technology. Part of the Adobe PDF Services API offering, it is available as a REST-powered SDK with underlying Sensei AI and is easily called by automated workflows.
+The [PDF Accessibility Auto-Tag API](pdf-accessibility-auto-tag-api/index.md) is a powerful tool that may be used as part of the process of improving accessibility in individual PDFs and can be applied at scale for large backlogs of PDFs. Auto-tagging adds document structure tags to the PDF file that are used to read a document's text and present it in a way that makes sense to users using assistive technology. Part of the Adobe PDF Services API offering, it is available as a REST-powered SDK with underlying Sensei AI and is easily called by automated workflows.
 
 Automatic tagging is a good first step on a document’s journey to full accessibility. It provides significant quality improvements over untagged PDF files including more accurate results for heading levels, reading order, complex lists, links, references, and tables. Screen reader users also benefit significantly from the improvements to heading levels for navigation.The service replaces any existing tags within your document, so it provides the most benefit for PDFs that have no tags or low-quality tags. An optional Excel report generated by the PDF Accessibility Auto-Tag service provides information related to the tagging of the document and helps to find content that you might need to review or correct.
 
 The output from the API is a tagged PDF, but it is not guaranteed to fully comply with accessibility standards such as WCAG and PDF/UA, as you may need to perform further downstream remediation and it is your responsibility to seek independent review of the same to meet those standards. After auto-tagging, to further improve accessibility, you may use Adobe Acrobat Pro to add alt-text to all figures and review the tagging of complex tables. Next you should check the first page and how the title is identified, the reading order, and then run the accessibility report, resolving any errors.
 
 You can learn more about PDF Accessibility in [this overview](https://www.adobe.com/accessibility/pdf/pdf-accessibility-overview.html).
 
+The PDF Accessibility Auto-Tag API can be embedded into any application using the [PDFServices SDK](./pdf-services-api/index.md#sdk) for Python and Java. [Start a PDF Accessibility Auto-Tag free trial that includes up to 1,000 free Document transactions or runs for 6 months, whichever occurs earlier.](https://documentservices.adobe.com/dc-integration-creation-app-cdn/main.html?api=pdf-accessibility-auto-tag-api)
```

Comment:
This will be available in all four variants, beta version was available in Java and Python only